src/Anexia.BDP.CloudLog/Client.cs

```diff
@@ -15,49 +17,47 @@ namespace Anexia.BDP.CloudLog
     /// </summary>
     public class Client
     {
-        private string brokers = "kafka0401.bdp.anexia-it.com:8443";
-        private string api = "https://api0401.bdp.anexia-it.com";
+        private const string api = "https://api0401.bdp.anexia-it.com";
 
-        private string index;
-        private string token;
-        private string caFile;
-        private string certFile;
-        private string keyFile;
-        private string keyPassword;
-        private string clientType;
-        private bool isHttp;
+        private static JsonSerializerOptions _jsonSerializerOptions = new()
+        {
+            PropertyNameCaseInsensitive = true
+        };
+
+        private readonly string _index;
+        private readonly string _token;
+        private readonly bool _isHttp;
         private long queue = 0;
 
-        private Producer<string, string> producer;
+        private string ClientType;
+
         private HttpClient httpClient;
 
+        private readonly Func<long> createTimeStamp =
+            () => (long)DateTime.Now.Subtract(DateTime.MinValue.AddYears(1969)).TotalMilliseconds;
+
         /// <summary>
-        ///     Creates a new client instance (kafka)
+        ///     Creates a new client instance (http)
         /// </summary>
         /// <param name="index">
         ///     Index name
         /// </param>
-        /// <param name="caFile">
-        ///     Path to ca file
-        /// </param>
-        /// <param name="certFile">
-        ///     Path to certificate file
+        /// <param name="token">
+        ///     Authentication token
         /// </param>
-        /// <param name="keyFile">
-        ///     Path to key file
+        /// <param name="client">
+        ///     Client for .net managed client capabilities
         /// </param>
-        public Client(string index, string caFile, string certFile, string keyFile, string keyPassword = "")
+        public Client(string index, string token, HttpClient client)
         {
-            this.index = index;
-            this.caFile = caFile;
-            this.certFile = certFile;
-            this.keyFile = keyFile;
-            this.keyPassword = keyPassword;
-            this.isHttp = false;
-            this.clientType = "dotnet-client-kafka";
-            CheckKafkaConfiguration();
-            CreateProducer();
+            _index = index;
+            _token = token;
+            _isHttp = true;
+            ClientType = "dotnet-client-http";
+            CheckHttpConfiguration();
 
+            httpClient = client;
+            httpClient.DefaultRequestHeaders.TryAddWithoutValidation("Authorization", token);
         }
 
         /// <summary>
```

Comment:
I do not think we need to mention `"(http)"` here, as this is the only thing we support.